apps/meteor/app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -349,7 +349,8 @@ export class CustomOAuth {
 					user.services &&
 					user.services[serviceName] &&
 					user.services[serviceName].id === serviceData.id &&
-					user.name === serviceData.name
+					user.name === serviceData.name &&
+					user.email === serviceData.email
```

Comment:
Same issue as the other PR - once that one is merged it will conflict here and you can keep that PR's version